video/cloud-client/analyze/beta_snippets.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates face detection, label detection,
+explicit content, and shot change detection using the Google Cloud API.
+
+Usage Examples:
+    python beta_snippets.py boxes \
+    gs://python-docs-samples-tests/video/googlework_short.mp4
+
+    python beta_snippets.py \
+    emotions gs://python-docs-samples-tests/video/googlework_short.mp4
+
+    python beta_snippets.py \
+    transcription gs://python-docs-samples-tests/video/googlework_short.mp4
+"""
+
+import argparse
+
+from google.cloud import videointelligence_v1p1beta1 as videointelligence
+
+
+# [START video_face_bounding_boxes]
+def face_bounding_boxes(path):
+    """ Detects faces' bounding boxes. """
+    video_client = videointelligence.VideoIntelligenceServiceClient()
+    features = [videointelligence.enums.Feature.FACE_DETECTION]
+
+    config = videointelligence.types.FaceConfig(
+        include_bounding_boxes=True)
+    context = videointelligence.types.VideoContext(
+        face_detection_config=config)
+
+    operation = video_client.annotate_video(
+        path, features=features, video_context=context)
+    print('\nProcessing video for face annotations:')
+
+    result = operation.result(timeout=900)
+    print('\nFinished processing.')
+
+    # There is only one result because a single video was processed.
+    faces = result.annotation_results[0].face_detection_annotations
+    for i, face in enumerate(faces):
+        print('Face {}'.format(i))
+
+        # Each face_detection_annotation has only one segment.
+        segment = face.segments[0]
+        start_time = (segment.segment.start_time_offset.seconds +
+                      segment.segment.start_time_offset.nanos / 1e9)
+        end_time = (segment.segment.end_time_offset.seconds +
+                    segment.segment.end_time_offset.nanos / 1e9)
+        positions = '{}s to {}s'.format(start_time, end_time)
+        print('\tSegment: {}\n'.format(positions))
+
+        # There are typically many frames for each face,
+        # here we print information on only the first frame.
```

Comment:
This comment confused me a little, especially with the possibility of segments above.
"There are typically many frames for each face, since a face is in a segment of time" (I don't know if that helps clarify it or just makes it more confusing)

Could a face appear in multiple segments?

Like:
Face 1: (Segment 0:00 --> 0:05, 0:15 --> 0:25) then the face would be in all the frames between those segments.

Reply:
I slightly reworded this comment, please take a look.

For face detection, each face indeed is a segment.  If the same person appears in two segments of the same video, two separate faces will be returned, each with its own segment.